packages/testcontainers/src/wait-strategies/wait.ts

```diff
@@ -23,8 +23,8 @@ export class Wait {
     return new HealthCheckWaitStrategy();
   }
 
-  public static forHttp(path: string, port: number): HttpWaitStrategy {
-    return new HttpWaitStrategy(path, port);
+  public static forHttp(path: string, port: number, options: HttpWaitStrategyOptions = {}): HttpWaitStrategy {
```

Comment:
What I meant regarding exporting from the index, is that the type `HttpWaitStrategyOptions` is now part of the public API, because `Wait` is exported: https://github.com/testcontainers/testcontainers-node/blob/6a54b4718e60651b84afa766092f3fa65cd265c5/packages/testcontainers/src/index.ts#L21

But the type `HttpWaitStrategyOptions` is not exported, so we probably need to add it like:

```ts
export { HttpWaitStrategyOptions } from "./wait-strategies/http-wait-strategy";
```